File: Kapil RNO 39/Java/Assign6/src/Personal.java
package finance;

public class Personal extends Loan implements Taxable {


	public double GST(double emi) {
		
		return emi * 1.18;
	}
	public float getRate() {

		float rate;

		if(getPrinciple() <= 500000)
			rate = 15;
		else
			rate = 16;

		return rate;
	}

	public Personal(double principle, float period) {

		setPrinciple(principle);
		setPeriod(period);
	}

}

File: Vishwas RNO 40/DotNet/Assignment 5/DemoApp/finance/Personal.cs
namespace finance;

public class Personal : Loan , ITaxable
{
    public Personal(double principle, double period)
    {
        this.principle = principle;
        this.period = period;
    }

    public double GST(double emi)
    {
        return emi * 1.18;
    }

    public override double GetRate()
    {
        double rate;

        if(principle <= 500000)
            rate = 15;

        else
            rate = 16;

        return rate;
    }
}